src/Agent/NewRelic/Agent/Core/Metrics/MetricNames.cs

```diff
@@ -732,6 +732,31 @@ public static string GetSupportabilityAgentApi(string methodName)
             return SupportabilityAgentApiPs + methodName;
         }
 
+        // OpenTelemetry Bridge
+        private const string SupportabilityOtelBridgePs = SupportabilityPs + "Metrics" + PathSeparator + "DotNet" + PathSeparator + "OpenTelemetryBridge" + PathSeparator;
+        
+        public const string SupportabilityOtelBridgeEnabled = SupportabilityOtelBridgePs + "enabled";
+        public const string SupportabilityOtelBridgeDisabled = SupportabilityOtelBridgePs + "disabled";
+        public const string SupportabilityOtelBridgeGetMeter = SupportabilityOtelBridgePs + "getMeter";
+        
+        // Individual meter method supportability metrics
+        private const string SupportabilityOtelBridgeMeterPs = SupportabilityOtelBridgePs + "meter" + PathSeparator;
+        
+        public const string SupportabilityOtelBridgeMeterCreateCounter = SupportabilityOtelBridgeMeterPs + "CreateCounter";
+        public const string SupportabilityOtelBridgeMeterCreateHistogram = SupportabilityOtelBridgeMeterPs + "CreateHistogram";
+        public const string SupportabilityOtelBridgeMeterCreateUpDownCounter = SupportabilityOtelBridgeMeterPs + "CreateUpDownCounter";
+        public const string SupportabilityOtelBridgeMeterCreateGauge = SupportabilityOtelBridgeMeterPs + "CreateGauge";
+        public const string SupportabilityOtelBridgeMeterCreateObservableCounter = SupportabilityOtelBridgeMeterPs + "CreateObservableCounter";
+        public const string SupportabilityOtelBridgeMeterCreateObservableHistogram = SupportabilityOtelBridgeMeterPs + "CreateObservableHistogram";
+        public const string SupportabilityOtelBridgeMeterCreateObservableUpDownCounter = SupportabilityOtelBridgeMeterPs + "CreateObservableUpDownCounter";
+        public const string SupportabilityOtelBridgeMeterCreateObservableGauge = SupportabilityOtelBridgeMeterPs + "CreateObservableGauge";
+
+        // Additional debugging metrics for OpenTelemetry Bridge operations
+        public const string SupportabilityOtelBridgeInstrumentCreated = SupportabilityOtelBridgePs + "InstrumentCreated";
+        public const string SupportabilityOtelBridgeInstrumentBridgeFailure = SupportabilityOtelBridgePs + "InstrumentBridgeFailure";
+        public const string SupportabilityOtelBridgeMeasurementRecorded = SupportabilityOtelBridgePs + "MeasurementRecorded";
+        public const string SupportabilityOtelBridgeMeasurementBridgeFailure = SupportabilityOtelBridgePs + "MeasurementBridgeFailure";
```

Comment:
These metric names should probably have the prefix `SupportabilityOTelMetricsBridge` to clarify that they apply to the metrics bridge. We will have additional metrics for the tracing bridge, and those metrics should have a different prefix.